src/bloom.h

```diff
@@ -115,9 +115,6 @@ class CBloomFilter
 class CRollingBloomFilter
 {
 public:
-    // A random bloom filter calls GetRand() at creation time.
-    // Don't create global CRollingBloomFilter objects, as they may be
-    // constructed before the randomizer is properly initialized.
```

Comment:
note to other reviewers: The comment no longer applies after 05fde14e3afe6f7156ebb6df6cd0e3ae12635b89